concent_api/verifier/tests/test_integration_verification.py

```diff
@@ -47,28 +47,17 @@ class VerifierVerificationIntegrationTest(ConcentIntegrationTestCase):
 
     def setUp(self):
         super().setUp()
-        self.compute_task_def = self._get_deserialized_compute_task_def(
-            task_id='ef0dc1',
-            subtask_id='zzz523',
-        )
-
+        self.task_to_compute = self._get_deserialized_task_to_compute()
+        self.compute_task_def = self.task_to_compute.compute_task_def
         self.source_package_path = get_storage_source_file_path(
-            task_id=self.compute_task_def['task_id'],
-            subtask_id=self.compute_task_def['subtask_id'],
+            self.task_to_compute.subtask_id,
+            self.task_to_compute.task_id,
```

Comment:
Hardly recommend to use parameters to be sure that `subtask_id` is `subtask_id` because we had bug which depends on that `subtask_id` was there `task_id`